vector/v.info/parse.c

```diff
@@ -56,4 +63,17 @@ void parse_args(int argc, char **argv, char **input, char **field, int *history,
         *shell |= SHELL_REGION;
     if (topo_flag->answer)
         *shell |= SHELL_TOPO;
+
+    if (strcmp(format_opt->answer, "plain") == 0) {
+        *format_ptr = PLAIN;
+    }
+    else if (strcmp(format_opt->answer, "json") == 0)
+        *format_ptr = JSON;
+    else {
+        *format_ptr = SHELL;
+    }
+
+    if (*shell != 0 && *format_ptr == PLAIN) {
+        *format_ptr = SHELL;
+    }
```

Comment:
```shell
v.info map="elev_lid792_bepts@PERMANENT" format="shell"
```

gives nothing.

Something like this is missing:

```c
if (strcmp(format_opt->answer, "json") == 0){
        *shell |= SHELL_BASIC;
        *shell |= SHELL_REGION;
        *shell |= SHELL_TOPO;
}
```

Reply:
Sorry, perhaps I misunderstood whether you are finished or not. Just let me know.

Reply:
Yes, I was done with the changes. I think I forgot to test all the changes for shell format. Hence, I missed this bug. Looking into fixing it.

Reply:
@wenzeslaus should be fixed now. I have updated and tested it locally.

Reply:
Great, thanks!